internal/controllers/topology/cluster/patches/engine_test.go

```diff
@@ -476,6 +476,19 @@ func setupTestObjects() (*scope.ClusterBlueprint, *scope.ClusterState) {
 			Namespace: metav1.NamespaceDefault,
 		},
 		Spec: clusterv1.ClusterSpec{
+			Paused: false,
```

Comment:
Can we add a patch to the "Should apply JSON patches to InfraClus.." test case above who uses one of those vars?

Reply:
Just realized that this would have to be in json_patch_generator_test.go, but it's not really worth it.